velox/exec/tests/utils/TpchQueryBuilder.cpp

```diff
@@ -25,9 +25,6 @@
 namespace facebook::velox::exec::test {
 
 namespace {
-int64_t toDate(std::string_view stringDate) {
-  return DATE()->toDays(stringDate);
```

Comment:
are we also planning to remove the toDays() method from DateType? Or there are still legit use cases for that?

Reply:
I'd like to remove that as well. Need to make sure this won't break anything though.